src/sql/workbench/contrib/executionPlan/browser/executionPlanPropertiesViewBase.ts

```diff
@@ -222,6 +228,10 @@ export abstract class ExecutionPlanPropertiesViewBase extends Disposable impleme
 		this._headerContainer.appendChild(c);
 	}
 
+	public setSummary(c: HTMLElement): void {
+		this._summaryContainer.appendChild(c);
```

Comment:
You don't want to clear out the container first? What happens if we set the summary again later?